### PR TITLE
with mmfunctions

### PR DESCRIPTION
**Pull request by retokromer**
- use mmfunctions
- add version where missing
- add help where missing

### CODE REVIEW

**Comment by retokromer** [comment]
I cannot understand, and give up.

**Comment by retokromer** [comment]
Giving up!

**Comment by dericed** [comment]
should we cherry pick from here, much seems useful

**Comment by retokromer** [comment]
I’m splitting it in separate issues, as every time I update the full, it’s again conflicting…

File: verifylto [hunk 1]
```diff
@@ -42,8 +42,8 @@ shift $(( ${OPTIND} - 1 ))
 
 SOURCE_DIR="${1}"
 _report_to_db
-if [[ ! $(echo "${TAPE_SERIAL}" | grep -E "${TAPE_SERIAL_REGEX}") ]] ; then
-    _report -w "${TAPE_SERIAL} is not valid. Error: The tape id must be exactly 6 capital letters and/or numbers, possibly followed by 'L5', 'L6' or 'L7' specifying the LTO generation."
+if [[ ! $(echo "${TAPE_SERIAL}" | grep "${TAPE_SERIAL_REGEX}") ]] ; then
+    echo "${TAPE_SERIAL} is not valid. The tape id must be exactly 6 capital letters and/or numbers."
```
Comment (dericed) [inline verifylto]: what is the purpose of this commit? Is it to negate part of 75f734c?

Comment (retokromer) [inline verifylto]: Yes, because it has become conflicting at a certain point.

Comment (retokromer) [inline verifylto]: I’m updating the files individually, and opening distinct PR.

Comment (retokromer) [inline verifylto]: Once done I can add the new files.

**Pull request closed without merging**